pydecred/rpc.py

```diff
@@ -60,12 +60,183 @@ def getBlockchainInfo(self):
         """
         return GetBlockChainInfoResult.parse(self.call("getblockchaininfo"))
 
+    def searchRawTransactions(
+        self,
+        address,
+        verbose=1,
+        skip=0,
+        count=100,
+        vinextra=0,
+        reverse=False,
+        filteraddrs=[],
```

Comment:
~~filteraddrs~~ -> filterAddrs

Reply:
Also set this to None and make it an empty list in the __init__ method.